packages/utils/src/getUiOptions.ts

```diff
@@ -23,5 +25,5 @@ export default function getUiOptions<T = any, S extends StrictRJSFSchema = RJSFS
         return { ...options, ...value };
       }
       return { ...options, [key.substring(3)]: value };
-    }, {});
+    }, globalOptions);
```

Comment:
super simple!

Reply:
I know, right!

Reply:
I wonder if I should update line 22 as `{ ...options }` to avoid potentially corrupting `globalOptions` if the caller modifies it?

Reply:
Actually this is better I think
```suggestion
    }, { ...globalOptions });
```